installer/x86_64/install.sh

```diff
@@ -536,17 +536,29 @@ fi
 # Decompress the file for the file system directly to the partition
 if [ x"$docker_inram" = x"on" ]; then
     # when disk is small, keep dockerfs.tar.gz in disk, expand it into ramfs during initrd
-    unzip -o $ONIE_INSTALLER_PAYLOAD -d $demo_mnt/$image_dir
+    if $( unzip -l $ONIE_INSTALLER_PAYLOAD 2>/dev/null 1>/dev/null ); then
+		unzip -o $ONIE_INSTALLER_PAYLOAD -d $demo_mnt/$image_dir
+	else
+        tar -C $demo_mnt/$image_dir -xvf $ONIE_INSTALLER_PAYLOAD
```

Comment:
Do you add the new ONIE_INSTALLER_PAYLOAD format? Why need to handle the tar case?